### DISCUSSION TITLE
Disable key events, that change month/year

### DISCUSSION THREAD

**Discussion by mathiasgv** [gpbl/react-day-picker#1135] [Support]
There is no way to disable the default keyboard events, for navigating between months and years(UP / DOWN / LEFT / RIGHT)

It would be nice, to have a way, to disable/override the handleKeyDown and handleDayKeyDown, in DayPicker.js.

I am asking, because currently on my page, some users are using the arrow keys, to scroll down further on the page, sometimes while having the DayPicker open, and they end up being in a completely different year without noticing

**Comment by cruzlutor** [comment]
Same issue here, it would be great if we can disable/enable the key events for component navigation

**Comment by emroussel** [comment]
@gpbl I would also like to be able to overwrite/disable this and would be willing to implement it. What do you think about of a `disableKeyUpDown` (or similar) prop to the `DayPicker` component?
